Source/ACE.Entity/Enum/Properties/PropertyInt.cs

```diff
@@ -2,409 +2,409 @@
 {
     public enum PropertyInt : ushort
     {
-        Undef = 0,
-        ItemType = 1,
-        CreatureType = 2,
-        PaletteTemplate = 3,
-        ClothingPriority = 4,
-        EncumbranceVal = 5, // ENCUMB_VAL_INT,
```

Comment:
no issues with the alignment, it was the note `// ENCUMB_VAL_INT` that was lost.. ACE calls it EncumbranceVal, client and aclogview call it ENCUMB_VAL_INT